lib/t/accounts.ex

```diff
@@ -682,4 +684,24 @@ defmodule T.Accounts do
     |> order_by(desc: :last_active)
     |> Repo.all()
   end
+
+  def push_users_to_complete_onboarding do
+    start_date = DateTime.add(DateTime.utc_now(), -24 * 60 * 60)
+    finish_date = DateTime.add(DateTime.utc_now(), -24 * 60 * 60 - 60)
+
+    Profile
+    |> where([p], is_nil(p.story))
+    |> where([p], p.last_active <= ^start_date)
+    |> where([p], p.last_active >= ^finish_date)
```

Comment:
didn't figure how to use SQL fragment now() with correct timezone

Reply:
what timezone is correct?

Reply:
UTC?

Reply:
```sql
select now();
-- 2021-11-16 11:25:40.825731+00
```

```elixir
iex(1)> DateTime.utc_now 
~U[2021-11-16 11:25:48.669111Z]
```

Reply:
https://xyproblem.info

Reply:
thanks, the solution to this indeed turned out to be:
`psql`
`ALTER DATABASE t_test SET timezone TO 'UTC';`

Reply:
@orlovvlad95 once I merge this PR into master you will experience this issue as well when testing, so pls run the above commands in your shell